common/logging.go

```diff
@@ -0,0 +1,46 @@
+package common
+
+import (
+	"io"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// InitLogging sets up logger with
+// appropriate logging level and path for log file
+// return error in case wrong log level parcing
+func InitLogging(cfg *Config) error {
+
+	lvl, err := log.ParseLevel(cfg.LogLvl)
+	if err != nil {
+		return err
```

Comment:
JFYI, we use https://github.com/gravitational/trace when returning errors everywhere so it is possible to print a call stack on error:

```go
return trace.Wrap(err)
```

Reply:
Done